.halcyon/sandbox-extra-apps

```diff
@@ -0,0 +1,3 @@
+alex-3.1.3
+happy-1.19.4
+yesod-bin-1.4.3.1
```

Comment:
Would it not make sense to use the overall constraints when installing these packages?